packages/blocks/src/api/i18n-block.json

```diff
@@ -0,0 +1,17 @@
+{
+	"title": "block title",
+	"description": "block description",
+	"keywords": [ "block keyword" ],
+	"styles": [
+		{
+			"label": "block style label"
+		}
+	],
+	"variations": [
```

Comment:
@nosolosw and @jorgefilipecosta – this schema format is very flexible, it works with quite complex structures like the one for block variations.

Reply:
That's good news :)